fudy/NoteScreen/note_screen/StickNoteModel.h

```diff
@@ -32,12 +32,13 @@ class StickNoteModel : public QAbstractListModel
 	Qt::ItemFlags flags(const QModelIndex& index) const override;
 	virtual QHash<int, QByteArray> roleNames() const override;
 
-	Q_INVOKABLE bool setItemAt(int index, const StickItem& item);
 	Q_INVOKABLE void removedItemAt(int index);
```

Comment:
Do you mean remove or removed??

Reply:
I mean "remove", I apologise for that mistake.